src/components/SignUp/OrganizationInformation.tsx

```diff
@@ -312,7 +312,7 @@ class OrganizationInformation extends Component<Props, State, {}> {
                 <label htmlFor="orgWebsite" className="col-sm-3 col-form-label text-sm-right">Organization website</label>
                 <div className="col-sm-9">
                   <input
-                    type="url"
+                    type="text"
```

Comment:
why is this a text field and not a url field anymore?

Reply:
dont change it to text. its already fine as is

Reply:
Fixed